WcaOnRails/config/database.yml.travis

```diff
@@ -0,0 +1,9 @@
+test:
```

Comment:
These two files being identical is just a coincidence. I don't want travis to break if we change the local connection settings. Speaking of local settings: I usually don't commit `database.yml` to the repository, but provide a `database.yml.sample` since people might want to customize their database settings which gets annoying if the file you modify is checked into source control. I decided against removing it, though since it would probably break the vagrant box.